x/config/chooser_test.go

```diff
@@ -267,6 +266,42 @@ func TestChooserConfigurator(t *testing.T) {
 			},
 		},
 		{
+			desc: "HTTP",
+			given: whitespace.Expand(`
+				outbounds:
+					their-service:
+						unary:
+							http:
+								url: "https://service.example.com/rpc"
+								peer: "127.0.0.1"
+			`),
+			test: func(t *testing.T, c yarpc.Config) {
+				outbound, ok := c.Outbounds["their-service"]
+				require.True(t, ok, "config has outbound")
+
+				require.NotNil(t, outbound.Unary, "must have unary outbound")
+				unary, ok := outbound.Unary.(*http.Outbound)
+				require.True(t, ok, "unary outbound must be HTTP outbound")
+
+				transports := unary.Transports()
+				require.Equal(t, 1, len(transports), "must have one transport")
+
+				transport, ok := transports[0].(*http.Transport)
+				require.True(t, ok, "must be an HTTP transport")
+
+				require.NotNil(t, unary.Chooser(), "must have chooser")
+				chooser, ok := unary.Chooser().(*peer.Single)
+				require.True(t, ok, "unary chooser must be a single peer chooser")
+
+				dispatcher := yarpc.NewDispatcher(c)
+				assert.NoError(t, dispatcher.Start(), "error starting")
+				assert.NoError(t, dispatcher.Stop(), "error stopping")
+
+				_ = transport
+				_ = chooser
```

Comment:
Why? If you're not using the variables, you can just put `_`s there.

    _, ok := transports[0].(*http.Transport)